config/config_center_config.go

```diff
@@ -132,11 +132,19 @@ func (c *CenterConfig) toURL() (*common.URL, error) {
 // it will prepare the environment
 func startConfigCenter(rc *RootConfig) error {
 	cc := rc.ConfigCenter
-	strConf, err := cc.prepareEnvironment()
+	dynamicConfig, err := cc.GetDynamicConfiguration()
 	if err != nil {
-		return errors.WithMessagef(err, "start config center error!")
+		logger.Errorf("Start dynamic configuration center error, error message is %v", err)
+		return err
 	}
+	envInstance := conf.GetEnvInstance()
+	envInstance.SetDynamicConfiguration(dynamicConfig)
 
+	strConf, err := dynamicConfig.GetProperties(cc.DataId, config_center.WithGroup(cc.Group))
+	if err != nil {
+		logger.Warnf("Dynamic onfig center has started, but config may not be initialized, because %s", err)
+		return nil
```

Comment:
Why return nil here?

Reply:
https://github.com/apache/dubbo-go/issues/1559
You should not force the user to publish the configuration when starting the configuration center.